### PR TITLE
Create entry API endpoint

### PR DESCRIPTION
**Pull request by uldisrudzitis**
- [x] API endpoint
- [x] Update state after creating entry
- [x] Remove unused code in `update.php`

### CODE REVIEW

File: _api_app/app/Sites/Sections/Entries/SectionEntriesDataService.php [hunk 1]
```diff
@@ -509,6 +511,164 @@ public function delete()
...
+            ]),
+            $section_entry_count
+        );
+
+        // update direct content property
+        $siteSectionsDataService = new SiteSectionsDataService($this->SITE);
+        $sectionTagsDataService = new SectionTagsDataService($this->SITE, $this->SECTION_NAME);
+        $section_tags = $sectionTagsDataService->populateTags();
+        $has_direct_content = !$section_tags['allHaveTags'] ? 1 : 0;
+        $siteSectionsDataService->saveValueByPath(
```
Comment (vteivans) [inline _api_app/app/Sites/Sections/Entries/SectionEntriesDataService.php]: Why not use the same `SiteSectionsDataService`?

Comment (uldisrudzitis) [reply _api_app/app/Sites/Sections/Entries/SectionEntriesDataService.php]: Added a comment why: https://github.com/berta-cms/berta/pull/225/files#diff-94bb9a8e67f5f1a90c376f66f4ea67baR634

The same in `deleteEntry` method.

File: editor/src/app/sites/sections/entries/entries-state/section-entries.state.ts [hunk 2]
```diff
@@ -48,19 +48,58 @@ export class SectionEntriesState implements NgxsOnInit {
...
+            action.site,
+            response.section_order,
+            {
+              '@attributes': {
+                entry_count: response.entry_count
+              }
+            })
+          );
+
+          dispatch(new UpdateSiteSectionAction(
```
Comment (vteivans) [inline editor/src/app/sites/sections/entries/entries-state/section-entries.state.ts]: why two `UpdateSiteSectionAction`?

Comment (uldisrudzitis) [reply editor/src/app/sites/sections/entries/entries-state/section-entries.state.ts]: `updateSiteSection` method can update only one property at a time

**Pull request merged**